### PR TITLE
Fix all controller files to match style guide

### PR DESCRIPTION
**Pull request by siaw23**
All tests passing for changes made in the controller folder.

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, this looks pretty good -- have you manually reviewed the `application_controller.rb` to confirm that it's just moved everything around and that all methods still exist? It's pretty hard to follow the diffs.

**Comment by jywarren** [comment]
This seems to cover beyond the controller, including models and other things in `/app` -- but it also passes tests. Have you booted this up and tried it out manually? Thanks, this is really great!

**Comment by siaw23** [comment]
if the tests are to be trusted then everything is fine. and yes, i have checked everything manually...

**Comment by jywarren** [comment]
There's a minor change I just merged in - can you resolve it and then I can merge this in? Thanks!!!

**Comment by siaw23** [comment]
@jywarren i rebased to master and the new file broker the test...

**Comment by jywarren** [comment]
Just getting back from travel. I'll take a look, thanks for your hard work on this!

**Comment by jywarren** [comment]
I just resolved conflicts again... that test shouln't be failing, let me see. I wonder if some part of it was reverted back in the rebase?

**Comment by jywarren** [comment]
Yes, see how this passed in #1294? Hmm... I'll look tomorrow but hopefully this new rebase passes.

**Comment by jywarren** [comment]
OK, so somehow one of the changes we rebased over was reverted, breaking the test. I think we'll need to be extra careful checking through this to ensure that doesn't happen elsewhere. Let's see how this runs.

**Comment by siaw23** [comment]
yes i need to take more care. i think i accidentally rebased off an old change from master. sorry about that. i believe your commit fixes it. will take extra care now on. @jywarren

**Comment by jywarren** [comment]
Hi, can you confirm that there are no changes to the tests? I'll read through it carefully for that and then we can merge this -- exciting!

**Comment by siaw23** [comment]
hi Jeff, I never touch tests, otherwise it makes no sense. so no, no changes to tests :)

**Comment by jywarren** [comment]
OK, this looks great then! Thank you! @ryzokuken @icarito @ananyo2012 @publiclab/reviewers please note this very thorough change to conform with the Ruby Style Guide, touching 60 files.

**Comment by jywarren** [comment]
Thank you so much, @siaw23 -- much appreciated. Did you close your other PR, by the way?

**Comment by siaw23** [comment]
@jywarren yes i did close the other one cause there were too many changes at once...
